### PR TITLE
 Configure aliases while configuring git

### LINKED ISSUE #29: Add aliases for all elegant git executables
Using [git aliases](https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases), need to configure shortcuts in the following format: 

`git elegant push` is equivalent to `git epush` - use prefix `e` before the command.

The aliases have to be generated on the fly based on available commands.

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Closes #29

### CODE REVIEW

**Comment by AntonKarabaza** [comment]
@extsoft Lets merge #88 first.

File: src/main/git-elegant-configure [hunk 1]
```diff
@@ -13,6 +13,10 @@ _user_email_key="user.email"
 _user_email_default=$(git config --global "$_user_email_key")
 _user_email_message="your user email"
 
+_alias_key="alias."
+_alias_default="yes"
+_alias_message="configure aliases"
```
Comment (extsoft) [inline src/main/git-elegant-configure]: `configure aliases` is not very informative. Let's use something like `add git aliases for all 'elegant git' commands`.

File: src/main/git-elegant-configure [hunk 2]
```diff
@@ -29,15 +33,25 @@ _configure() {
...
+                fi
             fi
         done
     done
 }
 
-GLOBALS=(_core_comment_char _user_name _user_email)
-LOCALS=(_core_comment_char _user_name _user_email)
+GLOBALS=(_core_comment_char _user_name _user_email _alias)
+LOCALS=(_core_comment_char _user_name _user_email _alias)
```
Comment (extsoft) [inline src/main/git-elegant-configure]: We need only `global` config for this command.

**Comment by AntonKarabaza** [comment]
@extsoft Branch rebased and comments applied. Please, take a look.

**Comment by extsoft** [comment]
@rultor please merge.

**Pull request merged**